ci_framework/roles/artifacts/tasks/edpm.yml

```diff
@@ -4,20 +4,21 @@
     - not ansible_facts.services is defined
   ansible.builtin.service_facts:
 
-- name: Check for virtualized compute
+- name: Check for virtualized compute in Baremetal jobs
   when:
     - "'libvirtd' in ansible_facts.services"
   block:
     - name: List all of the existing virtual machines
       register: vms_list
+      become: true
       community.libvirt.virt:
         command: list_vms
         uri: "qemu:///system"
 
     - name: Filter out edpm vm
       ansible.builtin.set_fact:
         ssh_key_file: "{{ cifmw_artifacts_basedir }}/artifacts/edpm/ansibleee-ssh-key-id_rsa"
-        ssh_user: root
+        ssh_user: cloud-admin
         edpm_vms: >-
```

Comment:
eventually we should get the IP instead of the amount of hosts... That would allow to get more than one compute. For later.